app/templates/index.html

```diff
@@ -100,6 +100,13 @@ <h2>Sell services</h2>
       </div>
     {% endif %}
 
+      <div class="padding-bottom-small">
+        <p>
+          <a href="https://www.gov.uk/government/collections/digital-marketplace-sales" class="top-level-link" target="_blank">
```

Comment:
Looking at other external links on the homepage, we don't tend to open in a new tab. I guess it could be less accessible?

Reply:
Okey dokey, have updated